app/main/templates/workbench.html

```diff
@@ -5,147 +5,157 @@
+                                                        </td>
+                                                        <td>{{ current_series.series_datetime }}</td>
+                                                        <td>{{ current_series.created_at.format('YYYY-MM-DD HH:mm') }}</td>
+                                                        <td class="hazen-accordion-centre">{{ current_series.user.username }}</td>
+                                                        <!-- <td>num files</td> -->
+                                                        <td class="hazen-accordion-centre">
+                                                            {% if current_series.has_report %}
+                                                                <a href="{{ url_for('main.series_view', series_id=current_series.id) }}">result</a>
+                                                            {% else %}
+                                                                None
+                                                            {% endif %}
+                                                        </td>
+                                                        <td class="hazen-accordion-centre">
+                                                            <a href="{{ url_for('main.delete', series_id=current_series.id) }}">
+                                                                <img src="../../static/trash.svg">
+                                                            </a>
+                                                        </td>
+                                                    </tr>
+                                                {% endfor %}
+                                                </tbody>
+                                            </table>
+                                        </div>
+                                    </div>
                                 </div>
                             </div>
-                        </div>
+                        {% endfor %}
+                        {#                    </div>#}
                     </div>
-                    {% endfor %}
                 </div>
             </div>
-        </div>
-    </div>
 
-</br>
-
-
-<div class="row"> <!-- col-md-4 col-lg-3 pb-3 -->
-
-</div>
-</form>
-<!--
-    <nav aria-label="...">
-        <ul class="pagination justify-content-center">
-            <li class="page-item{% if not prev_url %} disabled{% endif %}">
-                <a class= "page-link" href="{{ prev_url or '#' }}" tabindex="-1">
-                    <span aria-hidden="true">&larr;</span> Newer acquisitions
-                </a>
-            </li>
-            <li class="page-item{% if not next_url %} disabled{% endif %}">
-                <a class= "page-link" href="{{ next_url or '#' }}">
-                    Older acquisitions <span aria-hidden="true">&rarr;</span>
-                </a>
-            </li>
-        </ul>
-    </nav>
--->
```

Comment:
I think we should add this back as in the future we might have multiple pages of studies uploaded already, so it's better to have a way to go back to older stuff (it just hasn't made a difference yet with only a handful of test datasets)